### PR TITLE
Adding rendering for shop=video_games

### LINKED ISSUE #3117: Add icon for shop=video_games
https://wiki.openstreetmap.org/wiki/Tag:shop%3Dvideo_games
2.6k uses

Using the disc design from shop=music with some sort of games console?
Just to give an idea, a very quick sketch: video_gamesdraft01png (needs a better design)

### PR DESCRIPTION
**Pull request by doktorpixel14**
Fixes #3117 

Changes proposed in this pull request:
- Adding rendering for shop=video_games with a new icon

Test rendering with links to the example places:

Way z17 in [Orange Walk Town](https://www.openstreetmap.org/way/411391995) (has no name):
video_games way z17
z18:
video_games way z18
Node z17 in [East Greenwich](https://www.openstreetmap.org/node/4969327777):
video_games node z17
z18:
video_games node z18

### CODE REVIEW

**Comment by kocio-pl** [comment]
The icon has two colors (cable is violet), could you fix it and make in monochromatic?

**Comment by doktorpixel14** [comment]
Yes, that's because I used a stroke for the cable. I guess, there is no way to make a stroke without a specific colour, or is it?

**Comment by kocio-pl** [comment]
I don't know, but make it black then probably, like the rest,that would make it monochrome.

**Comment by doktorpixel14** [comment]
I have [this new version](https://gist.github.com/doktorpixel14/49a338a5ec875fbf82a6c22c93d9c3c1) now, where I converted the cable into a path. Obviously, it didn't change anything on the rendering.

File: project.mml [hunk 1]
```diff
@@ -1481,7 +1481,7 @@ Layer:
                                'mobile_phone', 'motorcycle', 'musical_instrument', 'newsagent', 'optician', 'jewelry', 'jewellery',
                                'electronics', 'chemist', 'toys', 'travel_agency', 'car_parts', 'greengrocer', 'farm', 'stationery',
                                'laundry', 'dry_cleaning', 'beverages', 'perfumery', 'cosmetics', 'variety_store', 'wine', 'outdoor',
-                               'copyshop', 'sports', 'deli', 'tobacco', 'art', 'tea', 'coffee', 'tyres', 'pastry', 'chocolate', 'music', 'medical_supply', 'dairy') THEN shop
+                               'copyshop', 'sports', 'deli', 'tobacco', 'art', 'tea', 'coffee', 'tyres', 'pastry', 'chocolate', 'music', 'medical_supply', 'dairy', 'video_games') THEN shop
```
Comment (kocio-pl) [inline project.mml]: I would like to have some line breaks to make shop list more readable, probably before 'music'.

File: project.mml [hunk 2]
```diff
@@ -1597,7 +1597,7 @@ Layer:
...
```
Comment (kocio-pl) [inline project.mml]: I'd like to have a line break here too.

**Comment by kocio-pl** [comment]
Do you think you can make the final changes soon?

**Comment by doktorpixel14** [comment]
Should be fine now

**Comment by kocio-pl** [comment]
Great, thanks a lot!

**Pull request merged**